classes/controller/AdminController.php

```diff
@@ -4166,8 +4166,21 @@ protected function processBulkStatusSelection($status)
                 /** @var ObjectModel $object */
                 $object = new $this->className((int) $id);
                 $object->active = (int) $status;
-                $result &= $object->update();
+                $update_ok = $object->update();
```

Comment:
```suggestion
                $update_ok = (bool) $object->update();
```

It currently returns an integer :(

Reply:
Yep ! thanks